src/toast/toast.js

```diff
@@ -185,9 +186,13 @@ class Toast extends React.Component<ToastPropsT, ToastPrivateStateT> {
                 overrides={closeIconOverrides}
               />
             ) : null}
-            {typeof children === 'function'
-              ? children({dismiss: this.dismiss})
-              : children}
+            {typeof children === 'function' ? (
+              children({dismiss: this.dismiss})
+            ) : (
+              <Block display="flex" flex="1">
```

Comment:
Could we avoid using this additional wrapper? It's not in overrides so created an element that can't be customized.

Reply:
yeah, probably just add 'flex' to `Body` with `justifyItems: 'space-between'` not totally sure what's going on with the markup ordering in this component. I imagine re-order close icon and children would avoid the `order` css property